packages/react-server/src/ReactFlightCache.js

```diff
@@ -9,9 +9,22 @@
 
 import type {CacheDispatcher} from 'react-reconciler/src/ReactInternalTypes';
 
+function createSignal(): AbortSignal {
+  return new AbortController().signal;
+}
+
 export const DefaultCacheDispatcher: CacheDispatcher = {
   getCacheSignal(): AbortSignal {
-    throw new Error('Not implemented.');
+    if (!currentCache) {
+      throw new Error('Reading the cache is only supported while rendering.');
+    }
+    let entry: AbortSignal | void = (currentCache.get(createSignal): any);
+    if (entry === undefined) {
+      entry = createSignal();
+      // $FlowFixMe[incompatible-use] found when upgrading Flow
```

Comment:
Nit: I don't get why a FlowFixMe is necessary here. Is it because the `createSignal` call resets the type refinement? Would a const binding above fix it?

Reply:
Looks like it was added during a codemod to `getCacheForType` below, and then you copy pasted. Guessing it's easy to fix.

Reply:
This is just temporary until I add the lifetime to Flight next, and getCacheForType will be replaced by cache().